src/github/client/github-installation-client.ts

```diff
@@ -362,6 +364,20 @@ export class GitHubInstallationClient extends GitHubClient {
 		return response?.data?.data?.viewer?.repositories?.totalCount;
 	}
 
+	public async getDependabotAlertsPage(owner: string, repoName: string, perPage = 1, cursor?: string): Promise<getDependabotAlertsResponse> {
+
+		const response = await this.graphql<getDependabotAlertsResponse>(getDependabotAlerts,
+			await this.installationAuthenticationHeaders(),
+			{
+				owner,
+				repo: repoName,
+				per_page: perPage,
+				cursor
+			},
+			{ graphQuery: "getDeploymentsQuery" });
```

Comment:
Shouldn't this be `getDependabotAlerts`?

Reply:
Its a metric name. It should be `getDependabotAlerts`. Updated it.